spec/components/deferral_details/view_spec.rb

```diff
@@ -21,13 +21,25 @@ module DeferralDetails
       it "renders the deferral date" do
         expect(component).to have_text(date_for_summary_view(data_model.date))
       end
+
+      it "renders the itt start date" do
+        expect(component).to have_text(date_for_summary_view(data_model.itt_start_date))
+      end
+
+      context "when trainee did not start" do
+        let(:trainee_stub) { Trainee.new }
+
+        it "renders the deferred before starting" do
+          expect(component).to have_text("Trainee deferred before their ITT started")
```

Comment:
I wondering if we can use the translation text instead of hardcoding the string. It would require `strip_tags()` though.

Reply:
Fixed